### ISSUE TITLE
Exception: Input to paramify must be a 1-D array.

### ISSUE THREAD

**Issue by sjackman** [HIPS/Spearmint#89]
Optimizing a single variable gives the error message
```
  File "/gsc/btl/linuxbrew/Cellar/python/2.7.12_3/lib/python2.7/site-packages/spearmint/tasks/base_task.py", line 271, in paramify
    raise Exception('Input to paramify must be a 1-D array.')
Exception: Input to paramify must be a 1-D array.
```

**Comment by sjackman** [comment]
@mgelbart I recall your having a fix for this bug, Mike. Could you please post the patch?

**Comment by sjackman** [comment]
This issue is a duplicate of #32 and resolved by #66.
